src/hooks/coreSDK/safeCoreSDK.ts

```diff
@@ -44,16 +44,37 @@ export const initSafeSDK = async (
   chainId: string,
   safeAddress: string,
   safeVersion: string,
+  chainInfo: ChainInfo,
+  walletChainId: string,
```

Comment:
I would prefix them so it is clearer which `chainId` is from which source (opened Safe or connected wallet)
```suggestion
  safeChainId: string,
  safeAddress: string,
  safeVersion: string,
  safeChainInfo: ChainInfo,
  walletChainId: string,
```